thirdparty/boost/include/boost/smart_ptr/detail/sp_counted_base_clang.hpp

```diff
@@ -26,6 +26,16 @@ namespace boost
 namespace detail
 {
 
+#if defined(__clang__)
+# pragma clang diagnostic push
+#endif
+
+#if defined(__clang__) && defined(__has_warning)
+# if __has_warning( "-Wc11-extensions" )
+#  pragma clang diagnostic ignored "-Wc11-extensions"
+# endif
+#endif
+
```

Comment:
I'm not comfortable modifying a third-party code...

Reply:
It is applying boostorg/smart_ptr@066b398 from upstream